Source/Initialization/WarpXInitData.cpp

```diff
@@ -976,23 +976,21 @@ WarpX::InitLevelData (int lev, Real /*time*/)
     // The default maxlevel_extEMfield_init value is the total number of levels in the simulation
     if ((m_p_ext_field_params->B_ext_grid_type == ExternalFieldType::parse_ext_grid_function)
          && (lev > 0) && (lev <= maxlevel_extEMfield_init)) {
+
+        // TODO: raise error when EB is activated
```

Comment:
Why is this comment here? Why should an error be raised when EB is activated?

Reply:
This is because I realized that the code is probably incorrect when using MR with the EM solver (i.e. when we use the `cp` version of the fields). 

This is because we should also have `cp` and `fp` versions of the arrays `eb_update_E` / `eb_update_B`.

That being said, the problem already existed before this PR. (i.e. we should have had `cp` and `fp` version of `edge_length` and `face_area`).

For now, I cleaned up this comment and will instead open an issue with more explanation.